src/RadioTile/RadioTile.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+import CheckIcon from '@rsuite/icons/Check';
+import Stack from '../Stack';
+import { RadioTileContext } from '../RadioTileGroup/RadioTileGroup';
+import { useClassNames, useControlled, partitionHTMLProps } from '../utils';
+import { WithAsProps } from '../@types/common';
+
+export type ValueType = string | number;
+
+export interface RadioTileProps<T = ValueType>
+  extends WithAsProps,
+    Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'> {
+  /** Additional description, if needed*/
+  children?: React.ReactNode;
+
+  /** Specifies whether the radio is selected */
+  checked?: boolean;
+
+  /** Specifies the initial state: whether or not the radio is selected */
+  defaultChecked?: boolean;
+
+  /** Whether the Radio is disabled */
+  disabled?: boolean;
+
+  /** Label of the Radio tile */
+  label?: React.ReactNode;
+
+  /** Icon to be used */
+  icon?: React.ReactNode;
+
+  /** Name to use for form */
+  name?: string;
+
+  /** Value, corresponding to the value of the RadioTileGroup, to determine whether the */
+  value?: T;
+
+  /** Callback function with value changed */
+  onChange?: (value?: T, event?: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Radio components has a `checked` argument for `onChange` callback. Is is intentionally omitted on RadioTile?

Reply:
The `checked` argument only has meaning in the `onChange` event handler of a `Checkbox`. In `RadioTile`, its value is always true.  I wish to remove it.

Reply:
That makes sense.